llvm/test/MC/Disassembler/SBF/sbf-ldst-new.txt

```diff
@@ -0,0 +1,62 @@
+# RUN: llvm-mc --disassemble %s -triple=sbf-solana-solana -mattr=+mem-encoding,+pqr-instr \
+# RUN:                          | FileCheck %s --check-prefix=CHECK-NEW
+# RUN: llvm-mc --disassemble %s -triple=sbf-solana-solana -mattr=+alu32,+mem-encoding,+pqr-instr \
+# RUN:                          | FileCheck %s --check-prefix=CHECK32-NEW
+
+
+# CHECK-NEW: ldxb r6, [r1 + 17]
+# CHECK-NEW: ldxb r1, [r10 - 3]
+# CHECK-NEW: ldxh r8, [r2 + 32]
+# CHECK-NEW: ldxw r4, [r3 + 64]
+# CHECK-NEW: ldxdw r0, [r4 + 128]
+0x2c,0x16,0x11,0x00,0x00,0x00,0x00,0x00
+0x2c,0xa1,0xfd,0xff,0x00,0x00,0x00,0x00
+0x3c,0x28,0x20,0x00,0x00,0x00,0x00,0x00
+0x8c,0x34,0x40,0x00,0x00,0x00,0x00,0x00
+0x9c,0x40,0x80,0x00,0x00,0x00,0x00,0x00
+
+
+
+# CHECK32-NEW: ldxb w6, [r1 + 17]
+# CHECK32-NEW: ldxb w1, [r10 - 3]
+# CHECK32-NEW: ldxh w8, [r2 + 32]
+# CHECK32-NEW: ldxw w4, [r3 + 64]
+0x2c,0x16,0x11,0x00,0x00,0x00,0x00,0x00
+0x2c,0xa1,0xfd,0xff,0x00,0x00,0x00,0x00
+0x3c,0x28,0x20,0x00,0x00,0x00,0x00,0x00
+0x8c,0x34,0x40,0x00,0x00,0x00,0x00,0x00
+
+
+
+# CHECK-NEW: stxb [r1 + 17], r6
+# CHECK-NEW: stxb [r10 - 3], r1
+# CHECK-NEW: stxh [r2 + 32], r8
+# CHECK-NEW: stxw [r3 + 64], r4
+# CHECK-NEW: stxdw [r4 + 128], r0
+0x2f,0x61,0x11,0x00,0x00,0x00,0x00,0x00
+0x2f,0x1a,0xfd,0xff,0x00,0x00,0x00,0x00
+0x3f,0x82,0x20,0x00,0x00,0x00,0x00,0x00
+0x8f,0x43,0x40,0x00,0x00,0x00,0x00,0x00
+0x9f,0x04,0x80,0x00,0x00,0x00,0x00,0x00
+
+
+
+# CHECK32-NEW: stxb [r1 + 17], w6
+# CHECK32-NEW: stxb [r10 - 3], w1
+# CHECK32-NEW: stxh [r2 + 32], w8
+# CHECK32-NEW: stxw [r3 + 64], w4
```

Comment:
Missing `stxdw`

Reply:
`CHECK32` tests loads for the 32-bit register class, so `stxdw` isn't available here.

Reply:
Have a look at line 35.